### PR TITLE
Adding an "about" page to Hotails

### LINKED ISSUE #70: Create an "About" section on Hotails
## Expected Behavior

When a user, rather it's a dog owner or a daycare, clicks on the "about" option on the navbar, they'll get the information about Hotails, just like the information from the 1st demo, such as:
- Available daycare actions on Hotails
- Available dog owner actions on Hotails
- Etc

The "about" page URL should be LOCALHOST_IP_ADDRESS/about

## Current Behavior

When the "about" option is clicked, the main app redirect the user to user's homepage.

## Acceptance Criteria

Clicking on the "about" option on the navbar redirect the user to LOCALHOST_IP_ADDRESS/about, with the 1st demo's homepage.

### PR DESCRIPTION
**Pull request by OfirMatasas**
### Desired Outcome

Having an "about" section of Hotails, using the old homepage from the 1st demo, where the user can learn about the available actions on Hotails for a dog owner and for a daycare.

### Implemented Changes

When a user clicks on the "about" option on the navbar, they're redirected to LOCALHOST_IP_ADDRESS/about, and from it to the other sections on Hotails.
For this matter, I've edited the old homepage to a simpler HTML file, moved the old homepage to the "about" section, and replaced the homepage to "Coming soon..." announcement, that would be replaced by customed homepage for a dog owner and a daycare.

A picture of the "about" section:
Untitled

A picture of the new "homepage":
Untitled

### Connected Issue/Story

Resolves #70 #19

### CODE REVIEW

File: Hotails/urls.py [hunk 1]
```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homepage, name='homepage')
+    path('', views.homepage, name='homepage'),
+    path('about/', views.about, name='about'),
 ]
```
Comment (ErezCohenn) [inline Hotails/urls.py:24]: I think the URL of the about page should be recognized in `main/urls` and in `Hotails/urls` you should use the include option as @tamirmatok did in #62.
And in general, I think this PR depends on #62, that only if you logged in to the project you will be able to enter the about page.

**Review by ErezCohenn — CHANGES_REQUESTED** [review]
Overall LGTM
Good job @OfirMatasas 🚀 
take a look at the comment I left below

File: Hotails/urls.py (hunk 1 above)
Comment (OfirMatasas) [reply Hotails/urls.py:24]: Good idea, I'll change that.

Comment (Omeramsc) [reply Hotails/urls.py:24]: It's your decision which pages should be available to guests and which not (personally I think including the about is ok).
my point was that the navbar should be available on all pages, with different content according to the user status (logged-in\ not logged-in)

Comment (OfirMatasas) [reply Hotails/urls.py:24]: > It's your decision which pages should be available to guests and which not (personally I think including the about is ok). my point was that the navbar should be available on all pages, with different content according to the user status (logged-in\ not logged-in)

Yesterday Tamir made the changes in the login page, by adding an if statement for checking in the user is logged in or not, and presenting the navbar options as well. So it's all taken care of :)

**Review by tamirmatok — APPROVED** [review]
Great job overall @OfirMatasas .
The about page looks amazing .
Just one thing to consider: This PR consider homepage as entrypoint and as we know it won't be like that after #62 will merge , and "test_redirected_to_homepage" as it wrote won't be relevant . 
So as I see it we have 3 options :
1) This PR should branch out from #62 .
2) Not consider homepage as entry-point
3) Wait to see which PR will merge first and make the suitable changes . 
Personalliy I think #62 should merge first , but still this PR not depend on it (:

**Comment by OfirMatasas** [comment]
> Great job overall @OfirMatasas . The about page look amazing . Just one thing to consider: This PR consider homepage as entrypoint and as we know it won't be like that after #62 will merge , and "test_redirected_to_homepage" as it wrote won't be relevant . So as I see it we have 3 options :
> 
> 1. This PR should branch out from #62 .
> 2. Not consider homepage as entry-point
> 3. Wait to see which PR will merge first and make the suitable changes .
> 
> Personalliy I think #62 should merge first , but still this PR not depend on it (:

I think that because of the fact that this pull request is not depended on #62, I'll keep the things as they are, and according to the merges we'll make the changes :)

**Review by jeniaSakirko — APPROVED** [review]
LGTM

**Review by Omeramsc — APPROVED** [review]
LGTM

**Review by ErezCohenn — APPROVED** [review]
GJ @OfirMatasas !

**Review by ormergi — APPROVED** [review]
Looks good!

**Pull request merged**